documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,218 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-recovery-volume_{context}"]
+= Recovering a cluster from persistent volumes
+
+This procedure describes how to recover a deleted cluster from persistent volumes.
+
+The procedure assumes that the Topic Operator is deployed.
+In situations such as these, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+The `KafkaTopic` resources must be recovered before a cluster is started so that the topics are not deleted by the Topic Operator.
+
+If you do not use the the Topic Operator, you just need to recover the `PersistentVolumeClaim` (PVC) resources.
+
+.Before you begin
+
+In this procedure, it is essential that PVs are mounted into the correct PVC to avoid data corruption.
+A `volumeName` is specified for the PVC and this must match the name of the PV.
+
+For more information, see:
+
+* xref:ref-persistent-storage-{context}#pvc-naming[Persistent Volume Claim naming]
+* xref:ref-jbod-storage-{context}#jbod-pvc[JBOD and Persistent Volume Claims]
+
+NOTE: The procedure does not include recovery of `KafkaUser` resources, which must be recreated manually.
```

Comment:
@scholzj - do we want to say more about this?

Reply:
I would probably leave that for another PR to make the review process easier. In general, thsi is easier:
* User has to recreate all the KafkaUser resources
* If the user wants to keep the old passwords / certificates, they have to recreate the secrets with them before creating the KafkaUSer resources

ultimately, there is not risk of data loss here, so IMHO not a big deal.

Reply:
Okay. I've extended the note for now:

> NOTE: The procedure does not include recovery of `KafkaUser` resources, which must be recreated manually. If passwords and certificates need to be retained, secrets must be recreated before creating the `KafkaUser` resources.